packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -470,10 +471,19 @@ export class RunningSummarizer implements IDisposable {
 			let overrideDelaySeconds: number | undefined;
 			let summaryAttempts = 0;
 			let summaryAttemptsPerPhase = 0;
+			// Reducing the default number of attempts to defaultNumberofSummarizationAttempts.
+			const totalAttempts =
+				loggerToMonitoringContext(this.logger).config.getNumber(
```

Comment:
It might be better to replace the logger in thes constuctor with a monitoring context in the constructor, similar to how its done [here](https://github.com/microsoft/FluidFramework/blob/09fc1c07e95a82725aac5595d8edbc3e910454ad/packages/runtime/container-runtime/src/containerRuntime.ts#L1103).
Then you can use this.mc.logger to log.